src/Digdir.Domain.Dialogporten.Domain/Dialogs/Entities/DialogEntity.cs

```diff
@@ -66,21 +65,30 @@ public class DialogEntity :
 
     [AggregateChild]
     public List<DialogActivity> Activities { get; set; } = new();
-    
+
+    public void SoftDelete()
+    {
+        foreach (var dialogElement in Elements)
```

Comment:
Her må vi potensielt kalle IDeletableEntity metodene for å soft delete selve dialogen

Reply:
Mener du `ISoftDeletableEntity`?
Og med "metodene", mener du da de i `SoftDeletableExtensions`? 

De blir jo håndtert for dialogen i `HandleAuditableEntities`, med ` `changeTracker.HandleSoftDeletableEntities(utcNow);`. 

(Ev. tenkte du at DialogElement og skulle implementere `ISoftDeletableEntity`?)

Reply:
Dones